common/MonoAssetManagerAndBuiltinResourceApplication.hpp

```diff
@@ -10,6 +10,8 @@
 #include "nbl/this_example/builtin/CArchive.h"
 #endif
 
+#include "nbl\system\CMountDirectoryArchive.h"
+
```

Comment:
this should come in from `nabla.h` more specifically `nbl/system.h` (or def/decl if it has them)